css/style.css

```diff
@@ -1,26 +1,190 @@
-/* ----------------------- Main Styles -----------------------*/
+/* ----------------------- Styles -----------------------*/
 
 * {
   font-family: 'Roboto', arial, sans-serif;
   font-size: 14px;
+  box-sizing: border-box;
+  text-align: center;
 }
 
+nav,
+main,
 footer {
-  color: #70757a;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  align-content: center;
+}
+
+a {
+  color: #222;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+/* ----------------------- Nav Styles -----------------------*/
+
+nav {
+  height: 60px;
+  padding: 6px;
+}
+
+.nav-item {
+  display: flex;
+  color: #222;
+}
+
+.left-subitem {
+  margin: 0 5px;
+  padding: 5px;
+}
+
+.left-subitem:nth-child(1) {
+  margin-left: 15px;
 }
 
-/* ----------------------- Grid Styles -----------------------*/
+.right-subitem {
+  color: #000;
+  font-size: 13px;
+  font-family: arial, sans-serif;
+  padding: 0 15px;
+  margin: auto;
+}
+
+.right-subitem:first-child {
+  padding-right: 0;
+}
+
+.right-subitem:last-child {
+  padding: 0px 8px;
+}
 
-.container {
-  height: 100vh;
+.small-grid {
   display: grid;
-  grid-template-columns: 1fr;
-  grid-template-areas:
-    "nav"
-    "nav"
-    "footer";
+  grid-gap: 2px;
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.dot {
+  height: 4px;
+  width: 4px;
+  background-color: #5f6368;;
+  border-radius: 50%;
+  display: inline-block;
+}
+
+#profile-image {
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+  padding: 4px;
+  margin-top: 4px;
+}
+
+/* ----------------------- Main Styles -----------------------*/
+
+main {
```

Comment:
The main content area seems a little too high. It might be a top margin defined as a percent of the viewport height with some max?